module/documents/token.mjs

```diff
@@ -206,8 +207,7 @@ export default class TokenDocument5e extends SystemFlagsMixin(TokenDocument) {
   /** @inheritDoc */
   _onUpdate(data, options, userId) {
     const textureChange = foundry.utils.hasProperty(data, "texture.src");
-    const tokenRingChange = foundry.utils.hasProperty(data, "flags.dnd5e.tokenRings.enabled");
```

Comment:
`this.#subjectPath` doesn't seem to depend on the enabled state, so I just removed it.